packages/orchestration/src/examples/staking-combinations.contract.js

```diff
@@ -104,12 +135,16 @@ const contract = async (zcf, privateArgs, zone, { orchestrateAll }) => {
   );
 
   const orchFns = orchestrateAll(flows, {
+    contractState,
     makeCombineInvitationMakers,
     makeExtraInvitationMaker,
     flows,
     zcf,
+    zoeTools,
   });
 
+  const creatorFacet = prepareChainHubAdmin(zone, chainHub);
```

Comment:
since this is an example contract, please include a comment about why this is needed.